<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddressBook</class>
 <widget class="QWidget" name="AddressBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Add Contact</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="customerTab">
      <attribute name="title">
       <string>Customer</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QWidget" name="customerWidget" native="true">
         <layout class="QFormLayout" name="formLayout_2">
          <property name="horizontalSpacing">
           <number>10</number>
          </property>
          <property name="verticalSpacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="customerNamelabel">
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="customerNamelineEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="customerDesignationLabel">
            <property name="text">
             <string>Designation:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="customerDesignationLineEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Ledger</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="customerCompanyLabel">
            <property name="text">
             <string>Company:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="companyLineEdit"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="customerTelephoneLabel">
            <property name="text">
             <string>Phone</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="customerTelephoneLineEdit"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="customerEmailLabel">
            <property name="text">
             <string>Email:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="customerEmailLineEdit"/>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="locationLabel">
            <property name="text">
             <string>Location</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="locationLineEdit"/>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="mobileLabel">
            <property name="text">
             <string>Mobile</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QLineEdit" name="mobileLineEdit"/>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="pOBoxLabel">
            <property name="text">
             <string>PO Box</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QLineEdit" name="pOBoxLineEdit"/>
          </item>
          <item row="11" column="0">
           <widget class="QLabel" name="buildingLabel">
            <property name="text">
             <string>Building</string>
            </property>
           </widget>
          </item>
          <item row="11" column="1">
           <widget class="QLineEdit" name="buildingLineEdit"/>
          </item>
          <item row="12" column="0">
           <widget class="QLabel" name="streetLabel">
            <property name="text">
             <string>Street</string>
            </property>
           </widget>
          </item>
          <item row="12" column="1">
           <widget class="QLineEdit" name="streetLineEdit"/>
          </item>
          <item row="13" column="0">
           <widget class="QLabel" name="addressLineLabel">
            <property name="text">
             <string>AddressLine</string>
            </property>
           </widget>
          </item>
          <item row="13" column="1">
           <widget class="QLineEdit" name="addressLineLineEdit"/>
          </item>
          <item row="14" column="0">
           <widget class="QLabel" name="cityLabel">
            <property name="text">
             <string>City</string>
            </property>
           </widget>
          </item>
          <item row="14" column="1">
           <widget class="QLineEdit" name="cityLineEdit"/>
          </item>
          <item row="15" column="0">
           <widget class="QLabel" name="countryLabel">
            <property name="text">
             <string>Country</string>
            </property>
           </widget>
          </item>
          <item row="15" column="1">
           <widget class="QLineEdit" name="countryLineEdit"/>
          </item>
          <item row="16" column="0">
           <widget class="QLabel" name="customerAddressLabel">
            <property name="text">
             <string>Notes</string>
            </property>
           </widget>
          </item>
          <item row="16" column="1">
           <widget class="QTextEdit" name="notesTextEdit"/>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="faxLabel">
            <property name="text">
             <string>Fax</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QLineEdit" name="faxLineEdit"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="customerButtonBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="vendorTab">
      <attribute name="title">
       <string>Vendor</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QWidget" name="vendorWidget" native="true">
         <layout class="QFormLayout" name="formLayout">
          <property name="horizontalSpacing">
           <number>10</number>
          </property>
          <property name="verticalSpacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="vendorNamelabel">
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="vendorNamelineEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="vendorDesignationLabel">
            <property name="text">
             <string>Designation:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="vendorDesignationLineEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="vendorCompanyLabel">
            <property name="text">
             <string>Company:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="vendorTelephoneLabel">
            <property name="text">
             <string>Telephone:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="vendorTelephoneLineEdit"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="vendorEmailLabel">
            <property name="text">
             <string>Email:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="vendorEmailLineEdit"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="vendorAddressLabel">
            <property name="text">
             <string>Address:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QTextEdit" name="vendorTextEdit"/>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="vendorButtonBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="employeeTab">
      <attribute name="title">
       <string>Employee</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QWidget" name="employeeWidget" native="true">
         <layout class="QFormLayout" name="formLayout_3">
          <property name="horizontalSpacing">
           <number>10</number>
          </property>
          <property name="verticalSpacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="employeeNamelabel">
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="employeeNamelineEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="employeeDesignationLabel">
            <property name="text">
             <string>Designation:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="employeeDesignationLineEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="employeeTelephoneLabel">
            <property name="text">
             <string>Telephone:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="employeeTelephoneLineEdit"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="employeeEmailLabel">
            <property name="text">
             <string>Email:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="employeeEmailLineEdit"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="employeeAddressLabel">
            <property name="text">
             <string>Address:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QTextEdit" name="employeeTextEdit"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="employeeButtonBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
